### ISSUE TITLE
Certain posts continue to crash TUI

### ISSUE THREAD

**Issue by fluffycoffeegoat** [ihabunek/toot#495]
hey there! I know this bug got reported back in June, but the toot UI is still crashing when hovering over certain posts. I know it was said there would be a fix soon, any ETA on that?

**Comment by danschwarz** [comment]
The PR to fix this has been open 2 months; waiting on @ihabunek  to approve it and merge into master.

**Comment by ihabunek** [comment]
Fixed in 0.44.0
